plugins/autocertkit

```diff
@@ -560,6 +560,20 @@ def install_rpm(session, vm_ref, username, password, rpm):
     """install one rpm"""
     install_rpms(session, vm_ref, username, password, [rpm])
 
+def install_vf_driver(session, vm_ref, username, password, package, driver_name):
+    """Install VF driver rpm on a droid vm"""
+    if package:
+        install_rpm(session, vm_ref, username, password, package)
+
+    if driver_name:
+        cmd = "echo %s > /etc/modules-load.d/%s.conf" % (driver_name, driver_name)
+        run_command(session, vm_ref, username, password, cmd)
+
+def disable_net_dev_naming(session, vm_ref, username, password):
```

Comment:
I'm concerned this is going to be very distro specific.

Reply:
ACK requires NIC name is 'ethX'. If we don't disable then we need to change code many places.